examples/README.md

```
@@ -81,6 +81,24 @@ chat -> orchestrator agent -> response
 
 **Required env vars:** `ROCKETRIDE_OPENAI_KEY`, `ROCKETRIDE_ANTHROPIC_KEY`
 
+### n8n-roundtrip.pipe
+
+**Call an n8n workflow from a RocketRide pipeline** (pairs with `n8n-call-rocketride.workflow.json`).
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)